scripts/ci/test_extensions.sh

```diff
@@ -20,13 +20,13 @@ exit_code=0
 
 # azure-cli-ml: https://github.com/Azure/azure-cli-extensions/issues/826
 # azure-iot: https://github.com/Azure/azure-cli/pull/17456
-block_list='azure-cli-ml azure-iot'
+ignore_list='azure-cli-ml azure-iot fzf'
```

Comment:
Apparently `ignore` is a better word compared to `blocked`, just like `ignore` and `ignore-patterns` from pylint.

http://pylint.pycqa.org/en/latest/technical_reference/features.html